<launch>
    <arg name="gestures" default="/random_thing_2" />
	<node name="rosbag" pkg="rosbag" type="play" args="--loop $(find rpstone)/bags/rps.bag">
		<remap from="/rps_gestures" to="$(arg gestures)"/>
	</node>
</launch>